src/libraries/Microsoft.PowerFx.Core/Public/ExpressionError.cs

```diff
@@ -147,7 +147,7 @@ internal static IEnumerable<ExpressionError> New(IEnumerable<IDocumentError> err
             }
             else
             {
-                return errors.Select(x => ExpressionError.New(x)).ToArray();
+                return errors.Select(x => ExpressionError.New(x, CultureInfo.CurrentCulture)).ToArray();
```

Comment:
>CultureInfo.CurrentCulture [](http://example.com/codeflow?start=65&length=26)

Why this change? We want to avoid  using CultureInfo.CurrentCulture.  

We want localization to be lazy (at point of reading the message), not point of creating the message.

Reply:
.Net analyzers detect when an API exists with a CultureInfo and if so, we need to use it

Reply:
Not should we would want not to use it